infrastructure/oss/issue_html.go

```diff
@@ -99,18 +99,34 @@ func getDetailsHtml(issue snyk.Issue) string {
 
 func buildPolicyMap(additionalData snyk.OssIssueData) map[string]interface{} {
 	policy := map[string]interface{}{}
+	severityChange := additionalData.AppliedPolicyRules.SeverityChange
+	annotation := additionalData.AppliedPolicyRules.Annotation
 
-	if additionalData.AppliedPolicyRules.SeverityChange.OriginalSeverity != "" {
-		policy["OriginalSeverity"] = additionalData.AppliedPolicyRules.SeverityChange.OriginalSeverity
-		policy["NewSeverity"] = additionalData.AppliedPolicyRules.SeverityChange.NewSeverity
-		policy["Reason"] = additionalData.AppliedPolicyRules.SeverityChange.Reason
+	hasPolicy := severityChange.OriginalSeverity != "" || annotation.Value != "" || annotation.Reason != ""
+	hasUserNote := annotation.Value != ""
+	hasNotes := severityChange.Reason != "" || annotation.Reason != ""
+
+	if severityChange.OriginalSeverity != "" {
```

Comment:
could be `if severityChange`

Reply:
If I understood correctly, you are suggesting replacing this line:

```go
if severityChange.OriginalSeverity != "" {
```

with:

```go
if severityChange != (snyk.SeverityChange{}) {
```

This check won't validate if we have a case like:

```
severityChange.OriginalSeverity = ""
severityChange.NewSeverity = "low"
severityChange.Reason = "not used"
```

Probably we will never have a case where a`NewSeverity` is present but `OriginalSeverity` is not but I think that the original check communicates better how we populate the map.